### ISSUE TITLE
Can you walk me on how to convert a float32 to a fixed?

### ISSUE THREAD

**Issue by abhinavgoel95** [MatthieuCourbariaux/deep-learning-storage#4]
I am new to programming, and I don't completely understand how you converted float32 to fixed.
I am trying to implement a Neural Net with Keras, and Im not sure how to store the weights and inputs as fixed point to measure the changes in accuracy.

Thanks! :)
